Civi/Payment/PropertyBag.php

```diff
@@ -717,6 +756,9 @@ public function setCurrency($value, $label = 'default') {
    * @return string
    */
   public function getDescription($label = 'default') {
+    if (!$this->has('description')) {
+      return '';
+    }
```

Comment:
Add default for non "essential" property description = ''

There is no reason to fail if we don't have a description set. This makes it easier to use `getDescription()` when filling parameter arrays without having to check it's set first.